test/test_autograd.py

```diff
@@ -1093,6 +1093,15 @@ def test_requires_grad_factory(self):
                             self.assertIs(dtype, output.dtype)
                             self.assertEqual(1, output.get_device())
 
+    def test_attribute_deletion(self):                                                            
+        x = torch.randn((5,5), requires_grad = True)                                      
```

Comment:
Spaces before and after `=` would probably cause lint error, and so will no space after `,` in `(5,5)`.